test/topology_custom/test_tablets_migration.py

```diff
@@ -71,7 +71,8 @@ async def setup(self):
                 self.log = await manager.server_open_log(servers[2].server_id)
                 self.mark = await self.log.mark()
             elif self.stage in [ "allow_write_both_read_old", "write_both_read_old", "write_both_read_new", "use_new" ]:
```

Comment:
> Right now the test assumes that stage names as test names them and as
they are written into system.tablets match

I have difficulty parsing this sentence from the commit message.

Reply:
LGTM otherwise.

Reply:
> I have difficulty parsing this sentence from the commit message.

My bad :( Rephrased the commit message.